java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2704,6 +2705,21 @@ public T type(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Renames <code>key</code> to <code>newKey</code>.
+     *
+     * @see <a href="https://redis.io/commands/rename/">redis.io</a> for details.
+     * @param key The <code>key</code> to rename.
+     * @param newKey The new name of the <code>key</code>.
+     * @return If the <code>key</code> was successfully renamed, return <code>"OK"</code>. If <code>
+     *     key</code> does not exist, an error is thrown.
```

Comment:
For the Transaction docs, we prefix the description with "Command Response - " since we are describing what the transaction returns when its executed rather than what the function itself returns
```suggestion
     * @return Command Response - If the <code>key</code> was successfully renamed, return <code>"OK"</code>. If <code>
     *     key</code> does not exist, the transaction fails with an error.
```

Reply:
so right, i totally forgot about that